### PR TITLE
std.concurrency: remove false Ddoc comments

### PR DESCRIPTION
**Pull request by WalterBright**
Passing for the moment on whether or not rows of `////` are proper Phobos style.

### CODE REVIEW

File: std/concurrency.d [hunk 1]
```diff
@@ -407,9 +407,9 @@ public:
     assert(res == "Child responding");
 }
 
-//////////////////////////////////////////////////////////////////////////////
+// ////////////////////////////////////////////////////////////////////////////
```
Comment (DmitryOlshansky) [inline std/concurrency.d]: Just kill the row, keep it at `//`. It looked weird before it's way more weird now.

Comment (WalterBright) [inline std/concurrency.d]: I think the moth-eaten look is a bit cool, dontcha think? :-)

Comment (andralex) [inline std/concurrency.d]: no, it's quite subtle - please let's not do this. Thanks!

**Comment by andralex** [comment]
trivial, merged without due process

**Pull request merged**